config/v1/types_apiserver.go

```diff
@@ -39,6 +39,9 @@ type APIServerSpec struct {
 	// The values are regular expressions that correspond to the Golang regular expression language.
 	// +optional
 	AdditionalCORSAllowedOrigins []string `json:"additionalCORSAllowedOrigins,omitempty"`
+	// encryption allows the configuration of encryption types for encryption of resources at the datastore layer.
```

Comment:
nit: consider removing the reference to the types, it could be more generic (`APIServerEncryption` is a top-level resource) so that in the future we don't have to update it.